src/main/java/Wolox/training/models/Book.java

```diff
@@ -1,43 +1,57 @@
-package Wolox.training.models;
+package wolox.training.models;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
 
 import javax.persistence.*;
+import java.util.List;
 
 @Entity
+@ApiModel(description = "Represents books from the OpenLibraryApi")
 public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
+    @ApiModelProperty("Represents the book title.")
```

Comment:
You can also use the `ApiModelProperty` attribute `required` to indicate whether a field it is required or not. In this case, only the field **gender** it is not required
```suggestion
    @ApiModelProperty(value = "Represents the book title.", required = true)
```